app/deck/gateway/diff.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Show differences in configuration (`deck gateway diff`)
```

Comment:
```suggestion
title: Show differences in configuration
```
I'd only include `deck gateway diff` in the title if we were going to do that in all of these pages. Otherwise, it stands out as a bit random. Also, backticks don't work in titles.